lib/thermometer.js

```diff
@@ -968,7 +991,22 @@ function Thermometer(opts) {
       get: function() {
         return toFixed(this.celsius + CELSIUS_TO_KELVIN, 2);
       }
-    }
+    },
+    freq: {
```

Comment:
I realize this approach is a little weird; I'm [copying an approach already used in the `Sensor` component](https://github.com/rwaldron/johnny-five/blob/v0.10.10/lib/sensor.js#L232-L246).